app/src/main/java/com/zeapo/pwdstore/git/GitServerConfigActivity.kt

```diff
@@ -125,8 +128,14 @@ class GitServerConfigActivity : BaseGitActivity() {
                 .setCancelable(false)
                 .setPositiveButton(R.string.dialog_delete) { dialog, _ ->
                     try {
-                        localDir.deleteRecursively()
```

Comment:
This can block when there are too many files, so now I run it in the IO Dispatcher.